lib/skate_web/channels/vehicles_search_channel.ex

```diff
@@ -0,0 +1,76 @@
+defmodule SkateWeb.VehiclesSearchChannel do
+  use SkateWeb, :channel
+  use SkateWeb.AuthenticatedChannel
+  require Logger
+
+  alias Realtime.Server
+  alias Util.Duration
+
+  @impl SkateWeb.AuthenticatedChannel
+  def join_authenticated(
+        "vehicles_search:" <> _unique_id,
```

Comment:
Still thinking through what `_unique_id` might be on the frontend, I'm thinking `property`. It is needed because duplicate subscriptions to the same topic aren't allowed from a single socket, so to support subscribing to multiple searches at once, the topics will have to be different.

Reply:
Maybe as an alternative, the topic could be`vehicles_search: <> search_params` kind of like the existing search, and only accept the limit via message. I wasn't sure before how to access the topic name from within the `handle_in` callback, but now I see you can get the topic as a field from [the socket](https://hexdocs.pm/phoenix/Phoenix.Socket.html#module-socket-fields).

This would mean subscribing to a different topic when the text changes, though I'm not sure that is a problem for regular search results. Maybe it depends on whether or not these channels should also be used for autocomplete.

Reply:
Resolution from discussion: search params will be put back into the topic name. Autocomplete will be a different topic.